<?xml version="1.0"?>
<launch>
	<arg name="roomba"      default="roomba1" />
	<arg name="config_path" default="$(find place_recognition)/config" />
	
	<!-- equirectangular_convertor, rosbag -->
	<include file="$(find place_recognition)/launch/utils/equirectangular_convertor.launch" >
		<arg name="is_bag"               value="true" />
		<arg name="theta_s_mode"         value="0" />
		<arg name="is_theta_s_visualize" value="false" />
		<arg name="is_rviz"              value="true" />
	</include>

    <!-- image_creator -->
    <node pkg="place_recognition" type="image_creator" name="image_creator" output="screen" >
		<!-- remap -->
		<remap from="img_in" to="roomba1/equirectangular/image_raw" />
		
		<!-- params -->
        <param name="FILE_PATH" value="$(find place_recognition)/img/" />
    </node>

</launch>